spec/models/seller_spec.rb

```diff
@@ -5,15 +5,55 @@
   # ensure Seller model has a 1:m relationship with the MenuItem model
   # it { should have_many(:menu_items).dependent(:destroy) }
   # Validation tests
-  let!(:seller) { Seller.create(seller_id: 'oiawjefoiwjaef') }
+  # let!(:seller) { create :seller }
+
+  before { create :seller}
   it { should validate_uniqueness_of(:seller_id) }
   it do
     should allow_value(%w[true false]).for(:sell_gift_cards)
   end
   it do
     should allow_value(%w[true false]).for(:accept_donations)
   end
+
+  let(:seller) do
```

Comment:
I think you can actually do let! and delete the before. ! happens before the before. I could be wrong here